.java-version

```diff
@@ -1 +1 @@
-1.8
+17
```

Comment:
should it be 11 as that's the mininal JDK version supported at the moment.

Reply:
fixed 11